bolt-sidecar/bin/sidecar.rs

```diff
@@ -18,37 +18,22 @@ const BOLT: &str = r#"
 
 #[tokio::main]
 async fn main() -> eyre::Result<()> {
+    println!("{}", BOLT);
```

Comment:
Curious -- do you prefer displaying the logo as the first thing or after all startup checks passed successfully?

I like the idea of seeing the logo knowing I've done everything well :D but maybe that's just me

Reply:
I get that, but I didn't like having other logs before the logo, such as:

[info] telemetry initialized!
LOGO
[...] other logs

instead it should be:
LOGO
[...] all logs

Not a blocker ofc, let's just pick one :)

Reply:
nvm, let's merge this! We can modify it at a later time if we wish